test/rubygems/test_gem_commands_signout_command.rb

```diff
@@ -8,6 +8,7 @@ class TestGemCommandsSignoutCommand < Gem::TestCase
 
   def setup
     super
+    File.delete Gem.configuration.credentials_path if File.exist?(Gem.configuration.credentials_path)
```

Comment:
We no longer need to do this since now we're only creating this file for the tests that need it, and they should be cleaning it up after themselves.

Reply:
Right. Thanks.